typesetting/aegisub.md

```diff
@@ -58,31 +57,32 @@ overwrite those. When in doubt, ask someone with Aegisub experience.
 Settings can be accessed via *View \> Options* or with the hotkey *Alt +
 O*.
 
-![aegisub64\_2017-09-08\_01-50-08.png](images/cnvimage100.png)]
+![aegisub64\_2017-09-08\_01-50-08.png](images/cnvimage100.png)
 
-![aegisub64\_2017-09-08\_01-54-35.png](images/cnvimage101.png)]
+![aegisub64\_2017-09-08\_01-54-35.png](images/cnvimage101.png)
 
-![aegisub64\_2017-09-08\_01-54-59.png](images/cnvimage102.png)]
+![aegisub64\_2017-09-08\_01-54-59.png](images/cnvimage102.png)
 
-The Advanced \> Video options include two particularly important
+The *Advanced \> Video* options include two particularly important
 settings.
 
 1.  "Force BT.601" Checkbox. This option is enabled by default. For
```

Comment:
FWIW, this isn't true with line0's builds. Probably deserves a mention for 3.2.2 users regardless.